frontend/src/views/activity/Activity.vue

```diff
@@ -165,7 +165,7 @@ Displays a single activity
 import ContentCard from '@/components/layout/ContentCard.vue'
 import ApiTextField from '@/components/form/api/ApiTextField.vue'
 import ApiSelect from '@/components/form/api/ApiSelect.vue'
-import ContentNode from '@/components/activity/ContentNode.vue'
+import RootNode from '@/components/activity/ContentNode.vue'
```

Comment:
Ich glaube du hast vergessen, den Import anzupassen. Aktuell wird der neue `RootNode` nie verwendet.
```suggestion
import RootNode from '@/components/activity/RootNode.vue'
```

Reply:
Nice catch. Nur: Ich hab das ja alles getestet und es hat einwandfrei funktioniert. I'm very confused now...

**Update:** Wir haben ungewollt Component auto registration eingeschaltet. Sprich RootNode is bereits automatisch importiert und gewinnt gegenüber der manuellen component registration. Ich fixe es hier und den Rest schauen wir in #2360 an